project/Templates/procedure-template.rst

```diff
@@ -36,21 +36,21 @@ Overview
 
 This is a template for a procedure. It is performed by authorized and trained users.
 
-.. _Title-of-Procedure-Prerequisites:
+.. _Title-of-Procedure-Precondition:
 
-Prerequisites
-^^^^^^^^^^^^^
+Precondition
+^^^^^^^^^^^^
 
 .. This section should provide simple overview of prerequisites before executing the procedure; for example, state of equipment, telescope or seeing conditions or notifications prior to execution.
 .. It is preferred to include them as a bulleted or enumerated list.
 .. Do not include actions in this section. Any action by the user should be included at the beginning of the Procedure section below. For example: Do not include "Notify specified SLACK channel. Confirmation is not required." Instead, include this statement as the first step of the procedure, and include "Notification to specified SLACK channel." in the Prerequisites section.
```

Comment:
I find this sentence very confusing, please consider removing it or rephrasing it.

Reply:
👍  I removed. It wasn't necessary to put so many constraints.